### PR TITLE
Okba/initialize project

### PR DESCRIPTION
**Pull request by xOKBAx**
Initialize Backend and Frontend

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -158,3 +158,20 @@ cython_debug/
...
+
+# dependencies
+node_modules
+
+# testing
+coverage
+
+# production
+build
+
```
Comment (kiskiller0) [inline .gitignore:173]: hey what is this ?
me no like this!

Comment (xOKBAx) [reply .gitignore:173]: ay men like it

Comment (kiskiller0) [reply .gitignore:173]: ok

**Review by kiskiller0 — APPROVED** [review]
LGTM

**Review by kiskiller0 — COMMENTED** [review]
LGTM

**Pull request merged**